audio/FAudio/Portfile

```diff
@@ -18,24 +18,27 @@ long_description        an XAudio reimplementation that focuses solely on develo
 
 depends_lib-append      port:libsdl2
 
-checksums               rmd160  eca88c3b235e6815c7c17bf6392c2ffdc30d9a25 \
-                        sha256  5846d61f088cd7b4450e6424a21bc4cff8f6074ab0cee702586b2a3b784bfdf6 \
-                        size    1131873
+checksums               rmd160  e24c9e179d5a806ad63d324a63a4200f6a6b321e \
+                        sha256  8aef339e0ddef9ca963a2903407e6ed254dbf8427fdb2977df6f47e99047f700 \
+                        size    1131793
 
 # remove set deployment target and hard-coded RPATH setting
 patchfiles              patch-faudio-remove-deployment-target.diff
 
-configure.args          -DFFMPEG=OFF \
+configure.args          -DGSTREAMER=OFF \
                         -DBUILD_UTILS=OFF \
                         -DBUILD_TESTS=ON \
                         -DXNASONG=ON
 
-variant ffmpeg description "Use ffmpeg for additional xWMA support - note license changes" {
+variant wma description "Use gstreamer1-gst-libav for additional xWMA support - note license changes" {
     license                GPL-2+
-    depends_lib-append     path:lib/libavcodec.dylib:ffmpeg
-    configure.args-replace -DFFMPEG=OFF -DFFMPEG=ON
+    depends_lib-append     path:lib/libgstlibav.dylib:gstreamer1-gst-libav
```

Comment:
There's only one port that provides libgstlibav.dylib (gstreamer1-gst-libav) so you can just write `port:gstreamer1-gst-libav`. (There are two ports that can provide libavcodec.dylib (ffmpeg and ffmpeg-devel) which is why we wrote `path:lib/libavcodec.dylib:ffmpeg` so that the user could choose which one they wanted.)

Please adjust the whitespace in this variant so that indentations are at multiples of 4 spaces.

Reply:
Your correct macports don’t require this only my own system need that, I’ll change this.

I don’t think I change the indentations but I’ll change that also